_i18n/de/resources/moneropedia/clearnet.md

```diff
@@ -1,32 +1,33 @@
 ---
-tags: ["kovri"]
+tags: ["kovri", "Kovri"]
 terms: ["Clearnet"]
-summary: "The Internet in which anonymous overlay networks are built upon"
+summary: "Das Internet, auf welchem anonyme Overlay-Netzwerke aufbauen"
```

Comment:
```suggestion
summary: "Das Internet, auf welchem überlagerte anonyme Netzwerke aufbauen"
```

Reply:
I don't dislike to translate the English version to "überlagerte [...] Netzwerke", but I thought "Overlay-Netzwerk" is a used and in this case contextually understood term. I cannot really decide which way would be better, though your suggestion would add an automatic explanation. One thought: If we decide to translate it, should it be a "überlagertes Netzwerk" oder "überlagerndes Netzwerk"? "Überlagert" implies that this very network ist overlayed by something (at least for me).

Reply:
I guess it depends on the audience (who's reading Moneropedia?) - "Overlay-Netzwerk" may be a term that's used in the context of some networking technology, though it might not be self explanatory to someone less tech-savvy, especially if he doesn't speak English. I think you're right with "überlagernde anonyme Netzwerke" being better than "überlagerte", which does seem a bit ambiguous to me now too :) Pick whichever you like better..!

Reply:
Alright. I'll use the "überlagernd"-Version within the text, but I will add the term "Overlay-Netzwerk" at some passage/place in the text to make it clear (and also to not underestimate the dear readers :) ).